katdal/h5datav3.py

```diff
@@ -579,6 +579,7 @@ def _get_telstate_attr(self, key, default=None, no_unpickle=()):
         try:
             value = self.file['TelescopeState'].attrs[key]
             if value in no_unpickle:
+                # Telstate attributes were serialised via str() until 2016-11-29
```

Comment:
Maybe merge this comment into the last sentence of the docstring.

Reply:
Done.